sr_unix/ydbinstall.sh

```diff
@@ -82,12 +82,12 @@ dump_info()
     if [ -n "$gtm_overwrite_existing" ] ; then echo gtm_overwrite_existing " : " $gtm_overwrite_existing ; fi
     if [ -n "$gtm_prompt_for_group" ] ; then echo gtm_prompt_for_group " : " $gtm_prompt_for_group ; fi
     if [ -n "$gtm_sf_dirname" ] ; then echo gtm_sf_dirname " : " $gtm_sf_dirname ; fi
-    if [ -n "$gtm_tmp" ] ; then echo gtm_tmp " : " $gtm_tmp ; fi
+    if [ -n "$gtm_tmpdir" ] ; then echo gtm_tmpdir " : " $gtm_tmpdir ; fi
```

Comment:
Don't understand the gtm_tmp -> gtm_tmpdir change. Are we renaming old envvars? May as well just drop the gtm_tmp flavor and not replace it forcing a change to ydb_tmpdir it would seem?

Reply:
That was a source of confusion for me too here. In this script, gtm_tmp was not the environment variable gtm_tmp but a shell variable. To make sure that distinction is clear, I gave the shell variable a name that is not any environment variable name.